skills/declarative/calendarSkill/dialogs/ListDialog/ListDialog.dialog

```diff
@@ -191,6 +205,14 @@
             }
           ],
           "elseActions": [
+            {
+              "$kind": "Microsoft.SetProperty",
```

Comment:
I wonder if we should challenge this way to doing the flow design.

This makes complete sense to a programmer like us. But if we are shipping this to customers who are just dragging and dropping, then would they be overwhelmed and spend way too much time wondering why this is there in the first place?

Basically the way I think about this is - the dialog has a state - the current index, the number of pages required to display all the results. The various events/intents (still trying to use the right terms here) just trying to advance this count +1 and -1 until it hits either the beginning or end. Can we model it in such a way that the dialog's flow can demonstrate this?

Reply:
By the way, pagination is such an interesting reusable thing (I think). We may want to drill down on it some more.